docs/samples/Microsoft.ML.Samples/Static/FeatureSelectionTransform.cs

```diff
@@ -0,0 +1,122 @@
+{
+    public class FeatureSelectionTransformStaticExample
+    {
+        public static void FeatureSelectionTransform()
+        {
+            // Downloading a classification dataset from github.com/dotnet/machinelearning.
+            // It will be stored in the same path as the executable
+            string dataFilePath = SamplesUtils.DatasetUtils.DownloadBreastCancerDataset();
+
+            // Data Preview
+            //    1. Label							0=benign, 1=malignant
+            //    2. Clump Thickness               1 - 10
+            //    3. Uniformity of Cell Size       1 - 10
+            //    4. Uniformity of Cell Shape      1 - 10
+            //    5. Marginal Adhesion             1 - 10
+            //    6. Single Epithelial Cell Size   1 - 10
+            //    7. Bare Nuclei                   1 - 10
+            //    8. Bland Chromatin               1 - 10
+            //    9. Normal Nucleoli               1 - 10
+            //   10. Mitoses                       1 - 10
+
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // First, we define the reader: specify the data columns and where to find them in the text file. Notice that we combine entries from
+            // all the feature columns into entries of a vector of a single column named "Features".
+            var reader = TextLoader.CreateReader(ml, c => (
+                        Label: c.LoadBool(0),
+                        Features: c.LoadFloat(1, 9)
+                    ),
+                separator: '\t', hasHeader: true);
+
+            // Then, we use the reader to read the data as an IDataView.
+            var data = reader.Read(dataFilePath);
+
+            // Second, we define the transformations that we apply on the data. Remember that an Estimator does not trnasform data
+            // directly, but it needs to be trained on data using .Fit(), and it will output a Transformer, which can transform data.
+
+            // In this example we define a CountFeatureSelectingEstimator, that selects slots in a feature vector that have more non-default 
+            // values than the specified count. This transformation can be used to remove columns with too many missing values.
+            // We also define a MutualInformationFeatureSelectingEstimator that selects the top k slots in a feature 
+            // vector based on highest mutual information between that slot and a specified label. Notice that it is possible to 
+            // specify the parameter `numBins', which controls the number of bins used in the approximation of the mutual information
+            // between features and label.
+            var pipeline = reader.MakeNewEstimator()
+                .Append(r =>(
+                    FeaturesCountSelect: r.Features.SelectFeaturesBasedOnCount(count: 695),
```

Comment:
>SelectFeaturesBasedOnCount [](start = 52, length = 26)

Is this the correct way to show the static api? Or should I use MLContext to find the estimator?